### DISCUSSION TITLE
rcs-na-iphone-z-ios-18-2-nadzieje-przywrocone/

### DISCUSSION THREAD

**Discussion by giscus** [idarek/dariusz.wieckiewicz.org#129] [General]
# rcs-na-iphone-z-ios-18-2-nadzieje-przywrocone/

Wolny świat, wolne myśly, wolne słowa.

https://dariusz.wieckiewicz.org/rcs-na-iphone-z-ios-18-2-nadzieje-przywrocone/?utm_medium=rss&utm_source=rss&utm_campaign=rss

**Comment by spexpl** [comment]
A ja dalej nie wiem, czy mój Samsung z Vikings Mobilis (play) oraz Oranege korzysta z RCS czy nie :( I za chiny nie wiem jak ot sprawdzić.
